src/amo/components/CollectionManager/index.js

```diff
@@ -154,7 +155,22 @@ export class CollectionManagerBase extends React.Component<Props, State> {
     event: ElementEvent<HTMLInputElement | HTMLTextAreaElement>
   ) => {
     event.preventDefault();
-    this.setState({ [event.target.name]: event.target.value });
+    const { name, value = '' } = event.target;
```

Comment:
I think if `value` is either not defined or `null`, we can early return. What do you think?

Reply:
okay, and also can handle `value.trim()` in this condition.